enhancements/api-review/new-CRD-ManagedClusterVersion.md

```diff
@@ -0,0 +1,490 @@
+---
+title: managed-cluster-version-crd
+authors:
+  - "@2uasimojo"
+reviewers: # Include a comment about what domain expertise a reviewer is expected to bring and what area of the enhancement you expect them to focus on. For example: - "@networkguru, for networking aspects, please look at IP bootstrapping aspect"
+  - "@wking"
+  - "@jewzaam: Managed OpenShift"
+  - "@cblecker: Managed OpenShift"
+  - "@pvasant (or delegate): Clusters Service (OCM)"
+  - "@berenss (or delegate): ACM"
+  - "@csrwng (or delegate): Hypershift"
+approvers: # A single approver is preferred, the role of the approver is to raise important questions, help ensure the enhancement receives reviews from all applicable areas/SMEs, and determine when consensus is achieved such that the EP can move forward to implementation.  Having multiple approvers makes it difficult to determine who is responsible for the actual approval.
+  - "@LalatenduMohanty"
+api-approvers: # In case of new or modified APIs or API extensions (CRDs, aggregated apiservers, webhooks, finalizers). If there is no API change, use "None"
+  - "@deads2k"
+creation-date: yyyy-mm-dd
+last-updated: yyyy-mm-dd
+tracking-link: # link to the tracking ticket (for example: Jira Feature or Epic ticket) that corresponds to this enhancement
+  - https://issues.redhat.com//browse/HIVE-2366
+see-also:
+  - n/a
+replaces:
+  - n/a
+superseded-by:
+  - n/a
+---
+
+# ManagedClusterVersion CRD
+
+## Summary
```

Comment:
This summary looks similar to https://github.com/kubernetes/enhancements/tree/master/keps/sig-multicluster/4322-cluster-inventory .  Determining if usage is appropriate and how we would build extensions could assist both efforts.